app/webpacker/components/ContactsPage/ContactForm.jsx

```diff
@@ -47,7 +50,10 @@ export default function ContactForm({
   const contactFormState = useStore();
   const dispatch = useDispatch();
   const { formValues: { contactRecipient: selectedContactRecipient, userData } } = contactFormState;
-  const formRedirection = useMemo(() => getFormRedirection(contactFormState.formValues));
+  const formRedirection = useMemo(
+    () => getFormRedirection(contactFormState.formValues),
+    [contactFormState.formValues],
+  );
```

Comment:
This calculation isn't expensive and the const isn't used in any dependency arrays, so I don't see any reason to memo this.

Reply:
Okay removed.